lib/wprof/reporters/external_report.rb

```diff
@@ -0,0 +1,13 @@
+module Wprof
+  module Reporters
+    module ExternalReport
+      def external_report
```

Comment:
This method name it's king of redundant don't you think?:
```
...::ExternalReport.external_report
```
maybe we could rename this method to `generate` so it's clear what your intention with this method is

Reply:
Done.